package.json

```diff
@@ -230,7 +230,7 @@
 		"express": "^4.17.1",
 		"express-rate-limit": "^5.2.6",
 		"fflate": "^0.7.1",
-		"fibers": "4.0.3",
+		"fibers": "^5.0.0",
```

Comment:
Is this going to break CentOS 7 support?

Reply:
I hope not. I can make sure and test it though